pkg/fftypes/operation.go

```diff
@@ -32,6 +32,8 @@ var (
 	OpTypeDataExchangeBlobSend OpType = ffEnum("optype", "dataexchange_blob_send")
 	// OpTypeTokensCreatePool is a token pool creation
 	OpTypeTokensCreatePool OpType = ffEnum("optype", "tokens_create_pool")
+	// OpTypeTokensAnnounce is a broadcast of token pool info
+	OpTypeTokensAnnouncePool OpType = ffEnum("optype", "tokens_announce_pool")
```

Comment:
🔊  - like where you settled on the terminology here. Would be good to remember this as we evolve to other announcements, such as on-chain logic interfaces.